src/NCronJob/Registry/JobRun.cs

```diff
@@ -65,30 +68,30 @@ private JobRun(
     public bool IsOneTimeJob { get; set; }
     public object? Parameter { get; }
     public object? ParentOutput { get; set; }
-    public int JobExecutionCount => Interlocked.CompareExchange(ref jobExecutionCount, 0, 0);
```

Comment:
Are we sure we don't need JobExecutionCount  anymore?

Reply:
There was absolutely no use throughout the codebase.

Reply:
Perfect!